components/app.js

```diff
@@ -11,6 +12,7 @@ class App {
     }
     handleGetGradesSuccess(grades){
       this.gradeTable.updateGrades(grades);
+      this.pageHeader.updateAverage(Math.floor((grades[0].grade) + (grades[1].grade) + (grades[2].grade)) / 3)
```

Comment:
this will only work if there are exactly three grades. Can you make it generic so that it works no matter how many grades there are?